pkg/steps/multi_stage/testdata/zz_fixture_TestGeneratePods.yaml

```diff
@@ -103,6 +103,7 @@
       image: sidecar
       name: sidecar
       resources: {}
+      terminationMessagePolicy: FallbackToLogsOnError
```

Comment:
I bet this is the source of breaking

Reply:
I lost the bet.

Reply:
It's actually the new config I am specifically trying to get by bumping this that is causing the breaking changes.